app/src/main/java/com/dcrandroid/adapter/TransactionListAdapter.kt

```diff
@@ -97,10 +97,16 @@ class TransactionListAdapter(val context: Context, val transactions: ArrayList<T
             holder.icon.setImageResource(iconRes)
         }
 
-        holder.itemView.setOnClickListener {
+        holder.itemView.container.setOnClickListener {
             TransactionDetailsDialog(transaction).show(context)
         }
 
+        if (position == itemCount - 1) {
```

Comment:
Some rows don't have a divider line

![Screenshot from 2021-01-14 15-03-31](https://user-images.githubusercontent.com/25265396/104601094-1d269a00-567a-11eb-90dc-4b14b5a7b0a8.png)

Reply:
Fixed in recent commit.